### PR TITLE
new config searching path

### LINKED ISSUE #78: add installation-level config path
between `~/.kuberlr/kuberlr.conf` and system-wide kuberlr.conf, a kuberlr installation can load a config in the same directory with the `kuberlr` binary.

### PR DESCRIPTION
**Pull request by gzm55**
fix #78

### CODE REVIEW

**Comment by gzm55** [comment]
@flavio `$KUBERLR_CFG` is ready here

File: internal/config/config.go [hunk 1]
```diff
@@ -59,16 +58,18 @@ func (c *Cfg) GetKubeMirrorURL() (string, error) {
 }
 
 func mergeConfig(v *viper.Viper, extraConfigPath string) error {
```
Comment (flavio) [inline internal/config/config.go:60]: Please rename `extraConfigPath` to `cfgFile`

File: internal/config/config.go [hunk 2]
```diff
@@ -59,16 +58,18 @@ func (c *Cfg) GetKubeMirrorURL() (string, error) {
...
-	cfgFile := filepath.Join(extraConfigPath, "kuberlr.conf")
+	if len(extraConfigPath) == 0 {
+		return nil
+	}
```
Comment (flavio) [inline internal/config/config.go:63]: This can be dropped when get `cfgFile` as input parameter

File: internal/config/config_test.go
Comment (flavio) [inline internal/config/config_test.go:1]: Can you add a new unit test that covers the env variable case?

File: internal/config/paths_unix.go [hunk 3]
```diff
@@ -4,13 +4,15 @@
 package config
 
 import (
+	"os"
 	"path/filepath"
 
 	"github.com/flavio/kuberlr/internal/common"
 )
 
 var configPaths = []string{ //nolint: gochecknoglobals // arrays cannot be go constants
```
Comment (flavio) [inline internal/config/paths_unix.go:13]: nit, rename it to `configFiles`, these are no longer paths.

File: internal/config/paths_windows.go [hunk 4]
```diff
@@ -4,13 +4,15 @@
...
 
 import (
-	"github.com/flavio/kuberlr/internal/common"
 	"os"
 	"path/filepath"
+
+	"github.com/flavio/kuberlr/internal/common"
 )
 
 var configPaths = []string{
```
Comment (flavio) [inline internal/config/paths_windows.go:13]: nit, rename it to `configFiles`, these are no longer paths.

**Review by flavio — CHANGES_REQUESTED** [review]
Overall LGTM, I left some comments.

Please update also the README to mention the new environment variable.

File: internal/config/config_test.go
Comment (gzm55) [reply internal/config/config_test.go:1]: append an empty string to path array in `TestMergeConfigs` to test the case when the KUBERLR_CFG is unset.

**Comment by gzm55** [comment]
> Overall LGTM, I left some comments.
> 
> Please update also the README to mention the new environment variable.

@flavio new env is listed in the README

**Comment by gzm55** [comment]
@flavio any suggestion on this pr?

**Review by flavio — APPROVED** [review]
Sorry for the late response, it has been a busy week

**Pull request merged**